Sprint_Tests/stepanov_maksym.md

```diff
@@ -0,0 +1,11 @@
+1. Previously, he was well-versed in markup and working on top-notch technologies at the frontend. After a few years of break and deep depression, plus listening to heavy music, he is incompetent in modern trends and drinks a lot. If he had run over depression, he wouldn’t have any problem found a job with a relevant salary. 
+
+2. If he had a goal and reasonable salary expectations he would work on remote somewhere in USA or Bali with an opportunity to relocate to the Canary Islands and apply for Spanish citizenship. Since childhood, he dreamed of marrying a Spanish woman and early retirement and growing grapes on the coast of the Balearic Sea.
+
+3. If you don't know how a piece of code written by a colleague works, you will have to cover it with console.log and trace the entire logical chain. This is a time-consuming task.  If we had documentation for this feature or at least a ticket for implementation with the description of business logic, we would release the fix in a couple of hours. 
+
+4. I want you to not hesitate to express your thoughts in a foreign language in spite of the mistakes that may appear due to a lack of practice and irregular language classes. I want each of you to read the book in the original by the end of the year, the original language is of course English, and enjoy overcoming the language barrier in communication.
+
+5. At the previous week, she had finished feature implementation and started a new one, today she has already implemented most of the user interface. Backend had written before she started working on a project,  so she had the swagger documentation and minimal functional description. She hasn’t started to write unit tests.
+
```

Comment:
she finished (you have a time marker and no other action that preceeded another so no Perfect tesnses here)
so far - is a period of time from the past up till the present = so far she has implemented most of the UI
Backend had been written - Passive voice as somebody wrote it not the back end itself.
working on the project (we already know what kind of project we are discussing here)